### PR TITLE
DIS-166 Add the `requesterId` to all `GET` calls

### PR DESCRIPTION
**Pull request by jogura**
To aide clients in cases where the authn session may have expired, the response with contain a `requesterId` so clients can verify/validate that the user's id is what the client expected it to be. If there is a mismatch then the clients can prompt the user to authenticate again.

@jimmordino @armonr

### CODE REVIEW

**Comment by jimmordino** [comment]
:+1:  Seems straightforward

File: service/discussion/src/main/java/com/wikia/discussionservice/mappers/HalAggregateRepresentationMapper.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package com.wikia.discussionservice.mappers;
```
Comment (armonr) [inline service/discussion/src/main/java/com/wikia/discussionservice/mappers/HalAggregateRepresentationMapper.java]: This files should be deleted. I think it came back in a merge

File: service/discussion/src/test/java/com/wikia/discussionservice/mappers/HalAggregateRepresentationMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
+package com.wikia.discussionservice.mappers;
```
Comment (armonr) [inline service/discussion/src/test/java/com/wikia/discussionservice/mappers/HalAggregateRepresentationMapperTest.java]: Same with this file. should be removed

Comment (armonr) [inline service/discussion/src/test/java/com/wikia/discussionservice/mappers/HalAggregateRepresentationMapperTest.java]: https://github.com/Wikia/pandora/pull/309

**Pull request merged**